src/test/groovy/org/prebid/server/functional/testcontainers/PbsServiceFactory.groovy

```diff
@@ -22,10 +24,12 @@ class PbsServiceFactory {
     }
 
     PrebidServerService getService(Map<String, String> config) {
+        final def containerCount = getContainersCount()
```

Comment:
1. No need to declare a var here, it is going to be used only once in that method, inline var pls.
2. Not clear what that method should return from its name: whether it is current container count or max container count, rename pls.
3. In groovy, if method name starts with get* and you are not going to provide parameters in that method call, you can call a method just like that: 'containersCount'